docs/docs/external/getting-started/test-network/ledger-initialization.md

```diff
@@ -75,9 +75,24 @@ Finally, to prepare both `network1` and `network2` for interoperation, run:
 ```bash
 ./bin/fabric-cli configure all network1 network2
 ```
-Instead, if you launched only one of the two Fabric networks, run the following after replacing `<network-id>` with either `network1` or `network2`:
+
+If Fabric networks were launched with 2 organizations, run:
+```bash
+./bin/fabric-cli configure all network1 network2 --num-orgs=2
+```
+
+Instead, if you launched only one of the two Fabric networks, run the following after replacing `<network-id>` with either `network1` or `network2`, and `<1/2>` with number of organizations in the network:
+```bash
+./bin/fabric-cli configure all <network-id> --num-orgs=<1/2>
+```
+
+Although IIN Agents can sync foreign network's membership automatically after every certain time period (by default 5 minutes), but **optionally**, fabric-cli can be used to trigger sync manually by running following command:
+```bash
+./bin/fabric-cli configure membership --local-network=network1 --target-network=network2 --iin-agent-endpoint=localhost:9500
+```
+This command syncs `network2`'s membership (target-network) in `network1` (local-network) using IIN Agent of `Org1MSP` as initiator. Similarly `network1`'s membership can be in `network2` by running:
 ```bash
-./bin/fabric-cli configure all <network-id>
+./bin/fabric-cli configure membership --local-network=network2 --target-network=network1 --iin-agent-endpoint=localhost:9501
 ```
```

Comment:
Add after this (in boldface) something like "Wait for at least 5 minutes before moving on to the next step (testing interoperability modes) to allow the networks' IIN Agents to sync their respective memberships."